Dockerfile

```diff
@@ -1,27 +1,42 @@
-FROM ubuntu:14.04
-MAINTAINER Shane Canon <scanon@lbl.gov>
+FROM eclipse-temurin:11.0.25_9-jdk-noble
 
-# Update apt and install jdk and docker engine to get docker clients
-RUN apt-get -y update && \
-    apt-get -y install openjdk-7-jdk make git ant && \
-    apt-get -y install apt-transport-https ca-certificates && \
-    apt-key adv \
-               --keyserver hkp://ha.pool.sks-keyservers.net:80 \
-               --recv-keys 58118E89F3A912897C070ADBF76221572C52609D && \
-    echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" > /etc/apt/sources.list.d/docker.list && \
-    apt-get -y update && apt-get -y install docker-engine=1.11.2-0~trusty 
+# Note if you update ubuntu the install lines below will need to be changed
+ENV CONTAINERD_VER=1.7.24-1
+ENV DOCKER_VER=27.4.1-1
+ENV DOCKER_BX_VER=0.19.3-1
+ENV DOCKER_COMPOSE_VER=2.32.1-1
 
-ADD . /src
+RUN apt update && apt install -y wget ant make git iptables
```

Comment:
Should this still have a ca-certificates update? Or unnecessary?

Reply:
I'm not entirely sure how that works to be frank. My vague understanding is that since this image is very new, all the certificates should be up to date any therefore the ca-certs update isn't needed

Reply:
I'm sure it's probably fine, then.